modin/experimental/core/storage_formats/hdk/query_compiler.py

```diff
@@ -558,14 +559,16 @@ def drop(self, index=None, columns=None):
             )
         )
 
-    def dropna(self, axis=0, how="any", thresh=None, subset=None):
-        if thresh is not None or axis != 0:
+    def dropna(self, axis=0, how="any", thresh=no_default, subset=None):
```

Comment:
To be aligned with api layer. Do we need to use default values here at all? 
```suggestion
    def dropna(self, axis=0, how=no_default, thresh=no_default, subset=None):
```

Reply:
@vnlitvinov perhaps this message was lost among many others :)

Reply:
@vnlitvinov, ping